htsohm/db/structure.py

```diff
@@ -0,0 +1,59 @@
+
+import sys
+import uuid
+
+from sqlalchemy import Column, ForeignKey, Integer, String, Float, Boolean
+from sqlalchemy.sql import text
+from sqlalchemy.orm import relationship
+
+from htsohm import config
+from htsohm.db import Base, session, engine
+from htsohm.db.atom_sites import AtomSites
+from htsohm.db.lennard_jones import LennardJones
+
+class Structure(Base):
+    """    """
+    __tablename__ = 'structure'
+
+    id = Column(Integer, primary_key=True)
+    lattice_constant_a = Column(Float)
+    lattice_constant_b = Column(Float)
+    lattice_constant_c = Column(Float)
+    
+    # relationship with 'materials'
+    material_id = Column(Integer, ForeignKey('materials.id'))
+    materials = relationship("Material", back_populates="structure")
```

Comment:
This is singular, right? 'materials' is always one material so should be named 'material'?